tests/test_util_file.expected.Darwin

```diff
@@ -0,0 +1,35 @@
+OK: json_object_from_fd(./valid.json)={ "foo": 123 }
```

Comment:
Bleh, that sucks about not being able to replace strerror(), but rather than ending up with per-arch output files, let's instead s/strerror/_json_c_strerror/ and then have the tests replace that.